src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<configuration>
+    <Appenders>
+        <console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{yyyy-MMM-dd HH:mm:ss} [%t] %-5level %-50c{1.} %msg %n">
+            </PatternLayout>
+        </console>
+        <File name="File" fileName="RedCord.log">
+            <PatternLayout pattern="%d [%t] %-5level %-50c{1.} - %msg %n"/>
+        </File>
+    </Appenders>
+    <Loggers>
+        <Root level ="error">
+          <AppenderRef ref="Console" />
+           <AppenderRef ref="File" />
+        </Root>
+    </Loggers>
```

Comment:
Logging is the process of providing information about an application as it performs different tasks or events. Logging offers benefits such as Issue Diagnosis and Analytics. With Log4j logging levels such as warning, error, fatal, debug etc. can be used, so that it will be easier to  read the logs and try to give them meaning to understand the importance of the message. 

Source: https://towardsdatascience.com/why-should-you-care-about-logging-442a195b80a1

Reply:
Good.